Modules/System/Translation/src/Translation.Codeunit.al

```diff
@@ -109,6 +109,34 @@ codeunit 3711 Translation
         TranslationImplementation.Delete(RecVariant, FieldId);
     end;
 
+    /// <summary>
+    /// Copies the translation for a field from on record to another record on a persisted (non temporary) record.
```

Comment:
A few typos here (e.g. from on record -> from one record). I'll fix those upon check-in.